health-services/project-factory/src/server/api/campaignApis.ts

```
@@ -1087,9 +1100,8 @@ async function enrichEmployees(employees: any[], request: any) {
   }
   const boundaryCodeAndBoundaryTypeMapping = getBoundaryCodeAndBoundaryTypeMapping(boundaryRelationshipResponse?.TenantBoundary?.[0]?.boundary);
   convertUserRoles(employees, request);
-  const idRequests = createIdRequests(employees);
-  request.body.idRequests = idRequests;
-  let result = await createUniqueUserNameViaIdGen(request);
+  const idRequests = createIdRequestsForEmployees(employees?.length, request?.body?.ResourceDetails?.tenantId);
+  let result = await createUniqueUserNameViaIdGen(idRequests);
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'result' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)